pages/glossary/semantic-role.md

```diff
@@ -13,6 +13,8 @@ The _semantic role_ of an element is determined by the first of these cases that
 1. **Explicit** If the element has an [explicit role][], then its _semantic role_ is its [explicit role][].
 1. **Implicit** The _semantic role_ of the element is its [implicit role][].
 
+This definition can be used in expressions such as "semantic `button`" meaning any element has a _semantic role_ of `button`.
```

Comment:
```suggestion
This definition can be used in expressions such as "semantic `button`" meaning "any element with a _semantic role_ of `button`".
```

Reply:
I'm also not too keen on the italic on the non-defining instance of the term (`_semantic role_`) (because I'm reading the italic with the semantics "defining instance", maybe I'm reading too much into it…)
```suggestion
This definition can be used in expressions such as "semantic `button`" meaning any element has a semantic role of `button`.
```